install/installer/traf_add_user

```diff
@@ -208,15 +208,26 @@ fi
 #=================================================
 # Cycle through all nodes to create known_hosts file, even if only one node
 echo "***INFO: Creating known_hosts file for all nodes"
+NODES_SHORT=""
 for node in $NODE_LIST
-do
-    sudo su $TRAF_USER --command "ssh -q -oStrictHostKeyChecking=no $node hostname"
-    if [ $? -ne 0 ]; then
-        echo "***ERROR: Unable to ssh to node $node"
-        exit -1
-    fi
+do 
+   newNode=$(ssh -q -n $node hostname -s) 
```

Comment:
Yes, the user needs to set up passwordless ssh and there is a check for that earlier in traf_config_check